scipy-1.0/rebuttal.tex

```diff
@@ -0,0 +1,164 @@
+% based on template:
+% https://tex.stackexchange.com/questions/15532/looking-for-cover-letter-template
+
+\documentclass[10pt,stdletter,dateno]{newlfm}
+
+% to adjust vertical spacing properties of
+% newlfm class see: https://tex.stackexchange.com/a/65533/43006
+%\unprtop{0mm}
+\topmarginsize{0mm}
+\topmarginskip{0in}
+\headermarginsize{0in}
+\headermarginskip{0in}
+\MinHead{0in}
+%\dateskipafter{0pt}
+%\dateskipbefore{0pt}
+\sigskipbefore{40pt}
+%\sigskipafter{0pt}
+\closeskipbefore{1pt}
+%\closeskipafter{0pt}
+%\addrfromskipbefore{0pt}
+\addrfromskipafter{0in}
+\addrtoskipbefore{0in}
+%\addrtoskipafter{0pt}
+\newlfmP{Headlinewd=0pt,Footlinewd=0pt}
+\newlfmP{sigsize=10pt}
+\bottommarginskip{10pt}
+\MinFoot{0pt}
+\leftmarginsize{0.95in}
+\rightmarginsize{0.95in}
+
+\usepackage{kpfonts}
+\usepackage{hyperref}
+\usepackage{url}
+
+\hypersetup{%
+  linkbordercolor=blue,% hyperlink borders will be blue
+  pdfborderstyle={/S/U/W 1}% border style will be underline of width 1pt
+}
+\widowpenalty=1000
+\clubpenalty=1000
+
+\namefrom{Ralf Gommers,
+          Matt Haberland,
+          and Tyler Reddy}
+\addrfrom{%
+    \today\\[10pt]
+    Ralf Gommers\\
+    Quansight Labs\\
+    The Netherlands\\
+    \texttt{ralf.gommers@gmail.com}\\
+    %phone number\\
+    \\
+    Matt Haberland\\
+    BioResource and Agricultural Engineering\\
+    California Polytechnic State University\\
+    San Luis Obispo, CA 93407, USA\\
+    \texttt{mhaberla@calpoly.edu}\\
+    %phone number\\
+    \\
+    Tyler Reddy\\
+    CCS-7 Applied Computer Science Group\\
+    Los Alamos National Laboratory\\
+    Los Alamos, NM 87545, USA\\
+    \texttt{treddy@lanl.gov}\\
+    %phone number\\
+}
+
+\addrto{%
+Dr. Rita Strack\\
+Nature Methods Editorial Office\\
+One New York Plaza Suite 4500\\
+New York, NY 10004, USA}
+
+\greetto{Dear Dr. Strack,}
+\closeline{Sincerely,}
+\begin{document}
+\begin{newlfm}
+
+We appreciate the careful review and helpful suggestions from you and the reviewers. Each comment is quoted and addressed below; we also link the relevant GitHub issues in which we discussed and implemented each suggestion. As requested, the corresponding changes are underlined in the revised manuscript.
+
```

Comment:
maybe add something like "... except for minor changes in the references section that couldn't be highlighted due to a technical issue" as @jarrodmillman suggested in #251.

Reply:
So far we're 2 to 1 in favor of just briefly mentioning the major additions to refs & saying they weren't highlighted due to a technical issue. I or you could probably come up with a list and do it manually, but let's not dwell on this for too long...:

- the new unit testing textbook citation
- the new OpenHub citation
- the new Nature News CI article citation
- any others?

Reply:
[Original bbl (df63951af6a12a424734ce8e0163daff23779766)](https://www.dropbox.com/s/va3mjthxqa4pkd9/bib_orig.bbl?dl=0)
[Current bbl](https://www.dropbox.com/s/teuwh4wl552ol89/bib_revised.bbl?dl=0)